### PR TITLE
[Cognitive Services - Personalzier] Package Page

### PR DESCRIPTION
**Pull request by diberry**

### CODE REVIEW

**Comment by diberry** [comment]
@tyclintw Please review.

File: sdk/cognitiveservices/Personalizer/src/Microsoft.Azure.CognitiveServices.Personalizer.csproj [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure Cognitive Services Personalizer Library</AssemblyTitle>
-    <Description>Provides API functions for consuming Microsoft Azure Cognitive Services Personalizer APIs.</Description>
+    <Description></Description>
+    <Description>
+<![CDATA[<h1>Cognitive Services - QnA Maker</h1>
```
Comment (tyclintw) [inline sdk/cognitiveservices/Personalizer/src/Microsoft.Azure.CognitiveServices.Personalizer.csproj]: This says 'QnA Maker'.

Comment (diberry) [reply sdk/cognitiveservices/Personalizer/src/Microsoft.Azure.CognitiveServices.Personalizer.csproj]: @tyclintw fix made - please sign off.

**Comment by diberry** [comment]
@AlexGhiondea I have verified the generated package works using NuGet package explorer. #sign-off

**Review by AlexGhiondea — APPROVED** [review]

**Comment by AlexGhiondea** [comment]
@diberry should we merge this change in?

**Comment by diberry** [comment]
@AlexGhiondea - this won't fix the problem. I'll close.

**Pull request closed without merging**